tests/end_to_end/server_test.py

```diff
@@ -1,14 +1,11 @@
 """
 Assists in executing tests with a running server on localhost
 """
-from __future__ import division
-from __future__ import print_function
-from __future__ import unicode_literals
 
 import unittest
 import subprocess
 
-import server as server
+from . import server as server
```

Comment:
What is the point of "as server"? It will be named server in either way.

Reply:
Yeah, it seems to be how it was before